maven-setup/action.yml

```diff
@@ -22,6 +22,19 @@ inputs:
     description: 'Job matrix; this should not be overridden by users (INTERNAL).'
     required: false
     default: ${{ toJSON(matrix) }}
+  skip-pr-comment:
```

Comment:
can we define these variables with positives to avoid a double negative? Double negatives are confusing to reason about.
For example `write-pr-comment` with a default of `true`. Then you can set it to false to disable.
(same comment applies to all the other properties here)

Reply:
good point, let me align with the [gradle-setup action](https://github.com/gradle/actions/tree/main/setup-gradle#build-reporting) and name it `add-X`

Reply:
done